www/webapp/src/views/SignUp.vue

```diff
@@ -58,11 +58,11 @@
               <v-alert class="ml-8 mt-2" v-if="domainType == 'dynDNS'" type="info" outlined>
                 <p class="text-h6">Limitations of Domains Registered under {{ LOCAL_PUBLIC_SUFFIXES[0] }}</p>
```

Comment:
Correct capitalization:
Domains -> domains
Registered -> registered

Reply:
We use [title case](https://en.wikipedia.org/wiki/Title_case) (see main page)

Reply:
Fair enough, as long as it's‚ intentional 😉